mdw-common/build.gradle

```diff
@@ -93,6 +93,14 @@ dependencies {
     codegen group: "io.swagger", name: "swagger-codegen-cli", version: "2.3.1"
     codegen group: "io.limberest", name: "limberest", version: "1.2.5"
     
+    // markdown
```

Comment:
I think these should be compileOnly since you are treating these as dynamic dependencies.  Otherwise I'm afraid they will grow the size of the cli jar.